kubernetes/helm/pinot/README.md

```diff
@@ -273,8 +273,8 @@ helm install --namespace "pinot-quickstart"  --name kafka incubator/kafka --set
 #### Create Kafka topic
 
 ```bash
-kubectl -n pinot-quickstart exec kafka-0 -- kafka-topics --zookeeper kafka-zookeeper:2181 --topic flights-realtime --create --partitions 1 --replication-factor 1
-kubectl -n pinot-quickstart exec kafka-0 -- kafka-topics --zookeeper kafka-zookeeper:2181 --topic flights-realtime-avro --create --partitions 1 --replication-factor 1
+kubectl -n pinot-quickstart exec kafka-0 -- kafka-topics.sh --bootstrap-server kafka-0:9092 --topic flights-realtime --create --partitions 1 --replication-factor 1
+kubectl -n pinot-quickstart exec kafka-0 -- kafka-topics.sh --bootstrap-server kafka-0:9092 --topic flights-realtime-avro --create --partitions 1 --replication-factor 1
```

Comment:
do we have an idea which kafka version this was corresponding to. i think the previous definition was added 2 yrs ago and probably the kafka cli got changed.

Reply:
Well, since I'm new to this code base I can't be sure. However, as far as I know regarding Kafka they (as in both Apache Kafka and Bitnami) they have done changes to parameter names such as the bootstrap-server parameter.

Reply:
@xiangfu0 ^ do we need to have 2 version of the README instruction for different kafka? AFAIK we have 0.9 and 2.x support previously. 
also do we still need to keep support for older version of kafka? and if so how long?